website/src/get-started/quickstart-py.md

```diff
@@ -176,17 +176,20 @@ This runner is not yet available for the Python SDK.
 
 {:.runner-flink-local}
 ```
-This runner is not yet available for the Python SDK.
+Currently, running wordcount.py on Flink requires a full download of the Beam source code.
+See https://beam.apache.org/roadmap/portability/#python-on-flink for more.
```

Comment:
for all of these, I would suggest changing "for more." to  "for more information."